src/libs/actions/Travel.ts

```diff
@@ -32,10 +33,12 @@ function acceptSpotnanaTerms() {
 
     const error = new Error('Failed to generate spotnana token.');
 
+    const params: AcceptSpotnanaTermsParams = {domain};
+
     return new Promise((resolve, reject) => {
         asyncOpenURL(
             // eslint-disable-next-line rulesdir/no-api-side-effects-method
-            API.makeRequestWithSideEffects(SIDE_EFFECT_REQUEST_COMMANDS.ACCEPT_SPOTNANA_TERMS, null, {optimisticData, finallyData})
+            API.makeRequestWithSideEffects(SIDE_EFFECT_REQUEST_COMMANDS.ACCEPT_SPOTNANA_TERMS, params, {optimisticData, finallyData})
```

Comment:
Is it 100% absolutely necessary for this API command to use side effects? It seems to me like the `travelDotURL` can exist in Onyx, and then somewhere in the view component, listen for when the value goes from nothing to something, then do whatever navigation is necessary.

Reply:
@tgolen, I implemented this as a regular write command. I had to open a [Web-E PR](https://github.com/Expensify/Web-Expensify/pull/45210) to support that. Please have a look when you have time 🙏